### ISSUE TITLE
Library crashes when used with androidx.browser latest version

### ISSUE THREAD

**Issue by homer-jay** [GoogleChrome/android-browser-helper#36]
This library crashes when used with [androidx.browser](https://developer.android.com/jetpack/androidx/releases/browser) latest version (as of today, v1.2.0-alpha09).

### Steps to reproduce
1.  Configure the `app/build.gradle` with the latest versions of both androidx.browser and android-browser-helper:
```
api 'androidx.browser:browser:1.2.0-alpha09'
implementation 'com.google.androidbrowserhelper:androidbrowserhelper:0.1.0-alpha1'
```

2. Launch a Chrome Custom Tab like in the demo example:
```
TrustedWebActivityIntentBuilder builder = new TrustedWebActivityIntentBuilder(Uri.parse("https://www.google.com"))
	.setNavigationBarColor(Color.RED)
	.setToolbarColor(Color.RED);

new TwaLauncher(this).launch(builder, null, null);
```

### Expected result
The tab should open without any error. This is the case if we use androidx.browser version 1.2.0-alpha08.

### Actual result
An exception is thrown. Stack trace:
```
java.lang.NoSuchMethodError: No virtual method getUrl()Landroid/net/Uri; in class Landroidx/browser/trusted/TrustedWebActivityIntentBuilder; or its super classes (declaration of 'androidx.browser.trusted.TrustedWebActivityIntentBuilder' appears in /data/app/org.example.customtabsdemo-1/base.apk)
        at com.google.androidbrowserhelper.trusted.TwaLauncher.launchCct(TwaLauncher.java:139)
        at com.google.androidbrowserhelper.trusted.TwaLauncher.launch(TwaLauncher.java:127)
        at org.example.customtabsdemo.android.ui.AppListActivity.showWebBrowser(AppListActivity.java:193)
        at org.example.customtabsdemo.android.ui.AppListActivity.openApplication(AppListActivity.java:171)
        at org.example.customtabsdemo.android.ui.AppListActivity.access$600(AppListActivity.java:40)
        at org.example.customtabsdemo.android.ui.AppListActivity$1.onRowClicked(AppListActivity.java:116)
        at org.example.customtabsdemo.android.ui.adapters.AppDescriptorAdapter.lambda$onBindViewHolder$1$AppDescriptorAdapter(AppDescriptorAdapter.java:92)
        at org.example.customtabsdemo.android.ui.adapters.-$$Lambda$AppDescriptorAdapter$lOIWUz164uqu7oQSxw09vNIC5-8.onClick(lambda)
        at android.view.View.performClick(View.java:5246)
        at android.view.View$PerformClick.run(View.java:21256)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:145)
        at android.app.ActivityThread.main(ActivityThread.java:6914)
        at java.lang.reflect.Method.invoke(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)
```

### Proposed fix
`TwaLauncher` should be modified so that it no longer calls `TrustedWebActivityIntentBuilder.getUrl()`. That method no longer exists. It was removed in androidx.browser v1.2.0-alpha09.

**Comment by andreban** [comment]
@PEConn, FYI
 
Building the library against `androidx.browser v1.2.0-alpha09` is breaking due to API changes on `androidx.browser`. 

We're working to fix this on 'android-browser-helper' and will release an 'alpha02' version that will be compatible with `alpha09` of `androidx.browser`.

@homer-jay 
As a note, If you are not directly using the API from `androidx.browser`, you probably don't need to import it (as android-browser-helper will pull the compatible version automatically). So it may be possible for you to just remove `api 'androidx.browser:browser:1.2.0-alpha09'` from your `build.gradle`.

**Comment by andreban** [comment]
This has been fixed on #48
